test/test_jax_interop.py

```diff
@@ -25,6 +24,67 @@ def f(a, b):
     torch.testing.assert_close(
         b, torch.sin(torch.ones(3, 3)) + 1, check_device=False)
 
+  def test_call_jax_grad(self):
+    """
+    Test that we can call a JAX function and retrive grad from PyTorch/XLA lazy
+    tensor tracing.
+    """
+    dev = xm.xla_device()
+    a = torch.randn((3, 3), device=dev, requires_grad=True)
+    b = torch.randn((3, 3), device=dev, requires_grad=True)
+
+    ##############pytorch function##############
+    def f_torch(a, b):
+      return torch.sum(torch.cos(a) + torch.sin(b))
+
+    def f_backward_torch(f, a, b):
+      out = f(a, b)
+      print(out)
+      out.backward()
+      return a.grad, b.grad
+
+    _a = a.clone().requires_grad_(True)
+    _b = b.clone().requires_grad_(True)
+    out_torch = f_torch(_a, _b)
+
+    _a.retain_grad()
+    _b.retain_grad()
+    out_grad_torch = f_backward_torch(f_torch, _a, _b)
+
+    torch_xla.sync()
+    out_torch.detach()
+    out_grad_torch = [g.detach() for g in out_grad_torch]
```

Comment:
If the only thing we're doing next is comparing these tensor against the call_jax version, then we probably don't need to detach either. Detach returns a new tensor and also stops the gradient flow. If we're no longer doing any gradient propagation, then it's useless.